mreg_cli/util.py

```diff
@@ -274,6 +275,7 @@ def result_check(result, type, url):
 
 
 def _request_wrapper(type, path, params={}, ok404=False, first=True, **data):
+    path = urllib.parse.quote(path, safe = "/")
```

Comment:
This will lead to `E251 unexpected spaces around keyword / parameter equals` if we do linting. Fix the whitespace?

Reply:
Fixed